README.md

```diff
@@ -2,20 +2,8 @@
 
 This repository contains a JSON schema for the EU Digital Covid Certificate.
 
-========
-
-PLEASE NOTE THAT THIS REPOSITORY IS WORK-IN-PROGRESS FOR THE EU EHealthNetwork TechIOP Sub-Group and forms the basis for upcoming JSON Schema versions.
-
-At some point a given version from this repository will be promoted to:
-
-https://github.com/eu-digital-green-certificates/ehn-dgc-schema
-
-which is where the versions for production apps for the DCCG are maintained.
-
-========
-
-**Schema version:** 1.2.1
-**Release date:** 2020-05-27
+**Schema version:** 1.3.0
+**Release date:** 2020-06-09
```

Comment:
Change to 2020-06-11 if release is tomorrow.

Reply:
yes will do - will finalize this once the reviews are completed, I'd rather not tempt fate ;)